ush/python/pygfs/task/analysis_calc.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+from logging import getLogger
+import netCDF4 as nc
+import os
+from pprint import pformat
+from pygfs.jedi import Jedi
+from wxflow import (AttrDict, FileHandler, Task,
+                    parse_j2yaml,
+                    to_timedelta, add_to_datetime, to_fv3time, to_isotime,
+                    logit)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class AnalysisCalc(Task):
```

Comment:
Both this and the other new class really should be called something like CubeAnalysisCalc or FV3AnalysisCalc, because they are specific to the FV3 cube and won't work for SOCA, or any other grid. It's not ATM because it will work for the aerosols and land surface since they are on the FV3 grid